packages/net-www/exlibs/vivaldi.exlib

```diff
@@ -69,11 +69,11 @@ vivaldi_src_unpack() {
 
 vivaldi_src_install() {
     edo mv usr/share/doc/{${PN},${PNVR}}
+    edo rmdir -p --ignore-fail-on-non-empty "usr/share/gnome-control-center/default-apps"
```

Comment:
Previously I assumed that `rm -rf` was just to remove empty dir about which Exherbo usually complain and require either explicitly marking keepdir or removing it.

If Vivaldi have some file there, why we don't want it to be installed like say Brave browser do?